riscv/tests/riscv.rs

```diff
@@ -531,7 +531,7 @@ fn features_with_options<T: FieldElement>(options: CompilerOptions) {
         &from_elf,
         &[expected.into()],
         None,
-        true,
+        false,
```

Comment:
executor complains about these

Reply:
mmm that is weird, its a pretty simple program

Reply:
yea it complains about a submachine with no columns, I feel like the fix would be simple but wanted to keep this PR going, can fix that separately